components/Layout/Page.tsx

```diff
@@ -42,6 +47,7 @@ export const Page = ({ children }: PageProps) => {
         <meta name="description" content="These are yummy recipes." />
         <meta name="keywords" content="food,appetizers,desserts,drinks" />
         <link rel="icon" href="/favicon.ico" />
+        <base href="/" />
```

Comment:
Q: Why use base here?

Reply:
That was something I added when first setting up pages, and is probably not needed. I'll remove.